src/error/Details.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for detailss.
+ */
+
+#ifndef SQUID_SRC_ERROR_DETAILS_H
+#define SQUID_SRC_ERROR_DETAILS_H
+
+#include "error/Detail.h"
+#include "mem/PoolingAllocator.h"
+
+#include <vector>
+
+/// Multiple details of a single error, in "canonical" order without duplicates.
+/// Our "canonical" order is the approximate detail discovery order (e.g., the
+/// order of Error::update() calls). This class isolates multi-detail storage
```

Comment:
```suggestion
/// Multiple details of a single error, in temporal order of discovery without duplicates.
/// The order/timing of Error::update() calls is used to approximate detail discovery time.
/// This class isolates multi-detail storage
```

Reply:
I committed your suggestion (b9d27b0), fixed description formatting (6f52d32), and then removed stale "canonical" references (c0b1ed3).